.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+*   @folio-org/stripes-force
```

Comment:
It looks like the GH team name is just `@folio-org/stripes`

Reply:
??? There _is_ a [stripes](https://github.com/orgs/folio-org/teams/stripes) team, but the team we are naming here is [stripes-force](https://github.com/orgs/folio-org/teams/stripes-force), which will soon be updated to just me and John.

Reply:
ignore the weird + in the code line, the formatting is messed up for some reason